### ISSUE TITLE
Add NPC-specific properties to Actor Sheet

### ISSUE THREAD

**Issue by JPMeehan** [MetaMorphic-Digital/draw-steel#18]
Currently, the Actor sheet only has sections applicable to generic creatures as well as Characters. It should support NPC specific properties.

**Comment by TysonJouglet** [comment]
I would like to contribute back to this effort. My main hurdle at the moment is how to test my changes. Do you use the Linux/NodeJS download of fvtt and a symlink to the data from the project folder? I am primarily an Oracle database dev so this is a new process for me.

**Comment by JPMeehan** [comment]
Yeah the node install imo works best but the symlink should work with any install method. Also, the symlink is only necessary for intellisense support, the project should work fine otherwise

**Comment by TysonJouglet** [comment]
Ok great! I forked the repo and plan on implementing surges and shields as a first contribution. 

After that I'll take a look at NPC properties.

This is mostly new to me so feel free to call out any weirdness.

**Comment by JPMeehan** [comment]
Looks like `subroles` should get renamed to `organizations`
